cmd/plugins_test.go

```diff
@@ -28,24 +31,59 @@ import (
 var initChan = make(chan struct{}, 256)
 var testDirRoot string
 
+const (
+	rootDir   string = "./"
+	prefixDir string = "plugin_test_tempdir"
+)
+
 // makeDirWithSharedObjects creates a new temporary directory containing files under the runtime directory
 // it compiles all .go files into shared object files with extension ext in the corresponding directory
 // It returns the root of the directory and a cleanup function.
-func makeDirWithSharedObjects(files map[string]string, ext string) (root string, cleanup func()) {
-	root, cleanup, err := test.MakeTempFS("./", "plugin_test_tempdir", files)
+//
+// Code is duplicated from MakeTempFS() due to https://github.com/open-policy-agent/opa/issues/1185
+func makeDirWithSharedObjects(files map[string]string, ext string) (tempRootDir string, cleanup func()) {
```

Comment:
[nit] Do we need the named return arguments? This function is quite long now, and the naked `return` makes you go hunt for what it potentially has returned... How about
```suggestion
func makeDirWithSharedObjects(files map[string]string, ext string) (string, func()) {
```
and explicitly returning values?

Reply:
There are really no guidelines in OPA as long a function should be, but I found this one:

https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

So, fixed it.